ts/src/time-profiler-inspector.ts

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright 2022 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {TimeProfile, TimeProfileNode} from './v8-types';
+import * as inspector from 'node:inspector';
+
+const session = new inspector.Session();
+session.connect();
```

Comment:
Shouldn't this have a corresponding `disconnect` call ?  Would it make sense to move it inside `startProfiling` ?

Reply:
Ya probably should do this 👍 see this comment https://github.com/google/pprof-nodejs/pull/227#discussion_r1007443301

Only thing I can think is that it may be less efficient, but looking at the V8 code I'm not sure if that's true. Would probably need to test it.